client-cli/src/main/java/au/org/consumerdatastandards/client/cli/ReferenceTest.java

```diff
@@ -40,18 +45,28 @@ public void autoValidate(@ShellOption(help = "enable or disable") String enableV
     }
 
     @ShellMethod("Validate json payload(s) against CDS")
-    public void validatePath(@ShellOption(value = "-f", help = "payload file or folder") String fileOrFolder) throws IOException {
+    public void validatePath(@ShellOption(value = "-f", help = "payload file or folder") String fileOrFolder,
+                             @ShellOption(value="-m", help = "model", defaultValue = "") String model) throws IOException {
```

Comment:
Just a though (no action needed) about the TAB usage conversation we had before. This is a case where tabs up to the code in the line can't be used. The IDE should be smart to insert tabs up until the current block indent and spaces afterwards to perform the sought alignment (of the `@ShellOption` annotation in this case). But better not to try and align the current line to something in the previous line.